Comp2024/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -89,8 +89,8 @@ private enum RotaryMode
   private static final double        kRotaryAngleHandoff   = -130.3;
   private static final double        kRotaryAngleDeployed  = 19.51;
 
-  private static final double        kRotaryAngleMin       = -179.27;
-  private static final double        kRotaryAngleMax       = 21.9;
+  private static final double        kRotaryAngleMin       = kRotaryAngleRetracted - 2;
```

Comment:
Can we make these "offsets" that define the min and max both be some even number of degrees--like 3.0--instead of something like 2.39. Negative and positive offsets should be about the same "few" (3.0) degrees since they serve the same purpose.